addons/func_godot/src/fgd/func_godot_fgd_entity_class.gd

```diff
@@ -22,6 +22,9 @@ var prefix: String = ""
 ## Key value pair properties that will appear in the map editor. After building the FuncGodotMap in Godot, these properties will be added to a Dictionary that gets applied to the generated Node, as long as that Node is a tool script with an exported `func_godot_properties` Dictionary.
 @export var class_properties : Dictionary = {}
 
+## Used to automatically apply properties to node.
+@export var override_node_properties : bool = false
```

Comment:
Should be ordered at the bottom, below descriptions, due to descriptions being both more important and having a more direct relationship to the entity definition, in addition to being "the next step" in constructing class properties.

Also, I think we should call this `auto_apply_to_matching_node_properties`, as it's both more descriptive and more accurate to what's happening. I'm not much a fan of the term "override" in this particular case, since we're setting the properties for the first time on the newly generated node.

Reply:
Well, it is overriding the default prefab or class. But I'm not opposed to changing the name or moving it.

Reply:
I'm putting `auto_apply_to_matching_node_properties` between meta properties and descriptions. IMHO, it should be as close to `class_properties` as possible since that is what it effects.